src/engine/World.cpp

```diff
@@ -35,8 +35,6 @@ WorldInstance::WorldInstance()
       m_BspTree(*this),
       m_PfxManager(*this)
 {
-    // Both games start at 8:00 in the morning
-    m_WorldInfo.setTimeOfDay(8, 00);
```

Comment:
Shouldn't put 00 here to not confuse people using the function for other times. Integer values with an extra 0 in front of them are seen as octal-numbers, thus everything over 7 would result in something different than intended.